kubernetes-extension/src/main/java/org/ballerinax/kubernetes/utils/KubernetesUtils.java

```diff
@@ -299,63 +299,86 @@ public static String resolveValue(String variable) throws KubernetesPluginExcept
         }
         return variable;
     }
-
+    
     /**
-     * Generate map by splitting keyValues.
+     * Generate array of string using a {@link BLangArrayLiteral}.
      *
-     * @param keyValues key value paris.
-     * @return Map of key values.
+     * @param expr Array literal.
+     * @return Convert string.
      */
-    public static Map<String, String> getMap(List<BLangRecordLiteral.BLangRecordKeyValue> keyValues)
-            throws KubernetesPluginException {
-        Map<String, String> map = new LinkedHashMap<>();
-        if (keyValues != null) {
-            for (BLangRecordLiteral.BLangRecordKeyValue keyValue : keyValues) {
-                map.put(keyValue.getKey().toString(), resolveValue(keyValue.getValue().toString()));
+    public static List<String> getArray(BLangExpression expr) throws KubernetesPluginException {
+        if (expr instanceof BLangArrayLiteral) {
```

Comment:
Here without using 'instanceof' we can check the node kind. 

```java
if (expr.getKind() == NodeKind.ARRAY_LITERAL_EXPR) {
```

Reply:
We can also invert the condition and return the error first.

Reply:
Fixed with https://github.com/ballerinax/kubernetes/pull/301/commits/66ec19a0fb6d97a36b6121d8c27adf89e2956468